test/socket.stream.js

```diff
@@ -49,16 +51,21 @@ describe('socket.stream', function(){
               httpMsg.socket._readableState.reading.should.be.true
             }
           }
-          
+
           //conventional node streams emit data events to process zmq stream response
           httpMsg.on('data',function (msg){
             msg.should.be.an.instanceof(Buffer);
-            String(msg).should.equal('<!DOCTYPE html><head><meta charset="UTF-8"></head>'
-              +'<body>'
-                +'<p>derpin over protocols</p>'
-              +'</body>'
-            +'</html>');
-            done();
+            var res = String(msg).split('\n');
```

Comment:
well depending on the zeromq version, (and I think this is something particular to ZeroMQ STREAM msg, and doubtful having to do with node's http module), we'll receive the response in two forms.. the first is a header or some manner of header and another is more like a typical inbound HTTP response.. with body and all. this hackery sets the stage for checking when we have something akin to the HTTP response w/ body